### ISSUE TITLE
How do we access files from outside Termux (f.e. with TouchVim)?

### ISSUE THREAD

**Issue by trusktr** [termux/termux-app#524]
I'd like to use [VimTouch](https://play.google.com/store/apps/details?id=net.momodalo.app.vimtouch.fullruntime) to edit files that are in my Termux home folder.

I tried to make a symlink in my storage pointing to a folder in Termux, but that seems restricted by Android.

I also tried moving the folder to storage, in order to link from Termux to the storage location, but the folder I'm interested in has internal symlinks, and storage doesn't support creation of symlinks.

Basically what I have is a `src` folder in `~` that contains git repos, and these repos are Node.js projects which contain symlinks inside `node_modules`, and they simply won't work outside of Termux.

Any ideas how to edit in these projects using an external app?

**Comment by RSCipher001** [comment]
I had similar problem and FAT32 SD-Card doesn't supoort symlinks so to complete my work I have been using SSH server to access files from $HOME folder and it works fine.

**Comment by azispratama92** [comment]
There is .bashrc file under your termux home directory. ![](https://i.imgur.com/7a7.png)

Its really usufull to acess my external sd card. Works and tested on Android Lolipop.

**Comment by azispratama92** [comment]
![](https://i.imgur.com/7JW5Idh.png)

**Comment by aquatix** [comment]
I'd like to know too; the last two comments are for the other way around; I'd like to be able to edit a file inside the termux directory from another Android app (think Quickedit Pro or something).

**Comment by Leask** [comment]
Having the same issue. Any help?

**Comment by dany-micheal** [comment]
@aquatix  you can use the text editor of SE FILE explorer to edit Termux files it work fine for me, the only issue is you've to create your file first off all with termux then edit with the text editor of SE FILE EXPLORER

**Comment by aquatix** [comment]
@dany-micheal And how do you access /data/data/com.termux/files/home from SE Explorer or Solid Explorer without root? As far as I know and can see, that directory is private for the termux app. Otherwise it would be the/a solution indeed :)

**Comment by dany-micheal** [comment]
@aquatix  till now the only way i seen to access this  directory with a file explorer  is to root your device then in the menu of SE Explorer click Root Explorer

**Comment by dany-micheal** [comment]
@aquatix  i just tried a news tips and it's working fine for me.
follow this tips to Access shared and external storage in termux  
https://wiki.termux.com/wiki/Internal_and_external_storage

then with the command 'cp -R '   you can copy and paste in  'storage' the termux directory you want to have access

let me know if it's work for you

**Comment by fadkeabhi** [comment]
You can use this 
https://github.com/fadkeabhi/rajefm

**Comment by lorenzoemmy** [comment]
Please help how can I move files into the termux default folder with rooting or atleast copy the files out and Change the folder of termux

**Comment by xeffyr** [comment]
> the termux default folder with rooting or atleast copy the files out

?

![01](https://user-images.githubusercontent.com/25881154/87851223-42229600-c8ff-11ea-9e75-edf4be276743.jpg)

![02](https://user-images.githubusercontent.com/25881154/87851226-45b61d00-c8ff-11ea-9b66-e88fbc3b2340.jpg)

**Comment by xeffyr** [comment]
Closing issue as access already possible through SAF-capable file managers.
